tests/index.js

```diff
@@ -1,3 +1,5 @@
+import 'babel-polyfill';
```

Comment:
This was necessary for the `async`/`await` syntax. We already have a indirect dependency on `babel-polyfill`, but I added it to the `package.json` anyway to be explicit.